lace/lace_data/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "lace_data"
-version = "0.1.2"
+version = "0.1.3"
```

Comment:
Not that I'm overly concerned--I don't think it hurts to bump it--but is there a reason to bump `lace_data`? The only change I see is to `criterion`, which is a `dev-dependency` anyway